test/client.js

```diff
@@ -993,17 +993,11 @@ describe('Api client', function () {
 
             let xhr = this.sandbox.useFakeXMLHttpRequest();
 
-            const fileObject = new File(['I am a file'], 'someFile.txt');
```

Comment:
Figured out why this doesn't work: IE and Edge don't support the `File` constructor. https://caniuse.com/#feat=fileapi